backend/tracim_backend/lib/core/content.py

```diff
@@ -2209,3 +2211,9 @@ def _allow_empty_label(self, content_type_slug: str) -> bool:
         ):
             return True
         return False
+
+    def get_authored_content_revisions_infos(self, user_id: int) -> AuthoredContentRevisionsInfos:
+        revision_count = func.count(ContentRevisionRO.revision_id)
+        workspace_count = func.count(ContentRevisionRO.workspace_id.distinct())
```

Comment:
same questions. look magic to me.

Reply:
It is not, when fighting with black I forgot the filter on `owner_id`. This is now re-added.